dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+- `255.6 - REPLACE-REPAIR BUIDLING - FACTORY - repair to existing building minor`
+- `256 - OTHER REMODELING`
+- `256.1 - OTHER REMODELING - elevators`
+- `256.2 - OTHER REMODELING - escalators`
+- `256.3 - OTHER REMODELING - handicap ramps`
+- `256.4 - OTHER REMODELING - awnings`
+- `256.5 - OTHER REMODELING - fire alarms`
+- `256.6 - OTHER REMODELING - security alarms`
+- `256.7 - OTHER REMODELING - fire alarms - Hood Suppression System`
+- `256.8 - OTHER REMODELING - fire alarms - Sprinkler for Fire`
+- `256.9 - OTHER REMODELING - fire alarms - Security alarms`
+- `271 - MAJOR IMPROVEMENT - WRECK`
+- `272 - MINOR IMPROVEMENT - WRECK`
+- `291 - MAJOR IMPROVEMENT - BURNOUT`
+- `292 - MINOR IMPROVEMENT - BURNOUT`
+- `311 - CARRIER - NEW MAJOR CONSTRUCTION`
+- `312 - NON-CARRIER NEW MAJOR CONSTRUCTION`
+- `331 - CARRIER - REMODELING`
+- `332 - NON-CARRIER-REMODELING`
+- `351 - CARRIER - REMODELING`
+- `352 - NON-CARRIER - REMODELING`
+- `371 - MAJOR IMPROVEMENT WRECK`
+- `372 - MINOR IMPROVEMENT - WRECK`
+- `391 - MAJOR IMPROVEMENT - BURNOUT`
+- `392 - MINOR IMPROVEMENT - BURNOUT`
+- `400 - EXEMPT PROPERTIES`
+- `411 - LEASEHOLD`
+- `500 - OFFICE PERMITS`
+- `510 - RECHECKS`
+- `511 - OCCUPANCY`
+- `512 - PRIOR YEAR PARTIAL ASSESSMENT`
+- `513 - NEW CONSTRUCTION`
+- `514 - ADDED IMPROVEMENTS`
+- `515 - MANUALLY ISSUED PERMIT`
+- `516 - PRIOR YEAR RECHECK`
+- `517 - DIVISION PERMIT`
+- `517.1 - DIVISION PERMIT - Petition`
+- `517.11 - DIVISION PERMIT - Petition Leasehold`
+- `517.2 - DIVISION PERMIT - Condominimum`
+- `517.3 - DIVISION PERMIT - Subdivision`
+- `517.4 - DIVISION PERMIT - Condo Removal`
+- `517.5 - DIVISION PERMIT - Petition Road Taking`
+- `517.6 - DIVISION PERMIT - Vacation`
+- `517.7 - DIVISION PERMIT - Dedication`
+- `517.8 - DIVISION PERMIT - Other`
+- `517.9 - DIVISION PERMIT - Mix Class`
+- `518 - CK BLDG. SF/UP`
+- `519 - CONVERT TO NEW MANUAL`
+- `520 - AUDIT DEPT INCREASE TO CHART`
+- `530 - RECENT I/C SALES`
+- `549 - 1985   HISTORICAL LANDMARK`
+- `550 - 1986   HISTORICAL LANDMARK`
+- `551 - 1987   HISTORICAL LANDMARK`
+- `552 - 1988   HISTORICAL LANDMARK`
+- `553 - 1989   HISTORICAL LANDMARK`
+- `554 - 1990   HISTORICAL LANDMARK`
+- `555 - 1991   HISTORICAL LANDMARK`
+- `556 - 1992   HISTORICAL LANDMARK`
+- `557 - 1993   HISTORICAL LANDMARK`
+- `558 - 1994   HISTORICAL LANDMARK`
+- `559 - 1995   HISTORICAL LANDMARK`
+- `560 - 1996   HISTORICAL LANDMARK`
+- `561 - 1997   HISTORICAL LANDMARK`
+- `562 - 1998   HISTORICAL LANDMARK`
+- `563 - 1999   HISTORICAL LANDMARK`
+- `564 - 2000   HISTORICAL LANDMARK`
+- `565 - 2001   HISTORICAL LANDMARK`
+- `566 - 2002   HISTORICAL LANDMARK`
+- `911 - OCCUPANCY CODE - WILL NOT BE USED`
+{% enddocs %}
+
+## permit_status
+
+{% docs column_permit_status %}
+Status of the permit in the Assessor's permit workflow.
+
+Possible values for this variable are:
+
+- null: No status assigned
+- `C`: Closed
+- `L`: Legacy
+- `M`: Manager review
+- `O`: Open
+- `P`: Pending
+- `R`: Recheck
```

Comment:
```suggestion
- `C - Closed`
- `L - Legacy`
- `M - Manager review`
- `O - Open`
- `P - Pending`
- `R - Recheck`
```

Reply:
Fixed in 7d94c97522a10d4303a21f3e5634ae39c567aa4b.